### PR TITLE
Add semantic-ui

### PR DESCRIPTION
**Pull request by cpetzold**

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/public/favicon.png">
     <title>Distense</title>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" />
+    <link href="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.12/semantic.min.css" rel="stylesheet" />
```
Comment (JohnAllen) [inline frontend/public/index.html]: If we have `semantic-ui` in both `node_modules` and in index.html -- that just speeds it up or why do that?

File: frontend/src/components/Layout.js [hunk 2]
```diff
@@ -1,76 +1,45 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
+import {
+  Container,
+  Divider,
+  Dropdown,
+  Grid,
+  Header,
+  Image,
```
Comment (JohnAllen) [inline frontend/src/components/Layout.js]: Should we import all of these now?  Not a big deal if so but not using all of them.

File: frontend/public/index.html (hunk 1 above)
Comment (cpetzold) [reply frontend/public/index.html]: You can use a loader to include the css, but this was just easier -- and if we serve from the cdn its possible that someone could have it browser cached when hitting our page.

Comment (JohnAllen) [reply frontend/public/index.html]: Ah I see -- that makes sense.  Probably lots will!

**Pull request merged**